lib/helpers.js

```diff
@@ -24,6 +24,11 @@ function finallyRejectsWithInitial(promise, fn) {
 
 const promiseUtils = require('promise-callbacks');
 
+//  A simple way to get up to 14 random hex digits, not cryptographic
+const randomHex14 = () => {
+  return Math.floor(Math.random() * Number.MAX_SAFE_INTEGER).toString(16);
```

Comment:
@hughsw as you noted, Math.random() is not cryptographically secure. Thus this implementation will be prone to auto-generated job IDs colliding when run at sufficient scale.

Could you revise using [crypto.randomBytes()](https://nodejs.org/docs/latest-v12.x/api/crypto.html#crypto_crypto_randombytes_size_callback) or equivalent instead?